packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,712 @@
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 5,
+			"defaultValue": 0
+		},
+		{
+			"#": "13",
+			"label": "Heating Stage 2 On Threshold",
+			"description": "Sets the delta from setpoint that stage 2 heating starts.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 7,
+			"defaultValue": 2
+		},
+		{
+			"#": "14",
+			"label": "Heating Stage 2 Off Threshold",
+			"description": "Sets the delta from setpoint that stage 2 heating stops. Stage 2 turns off at setpoint + Delta Stage 2.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0
+		},
+		{
+			"#": "15",
+			"label": "Aux Heat On Threshold",
+			"description": "Sets the delta from setpoint that stage 3 heating starts",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 8,
+			"defaultValue": 3
+		},
+		{
+			"#": "16",
+			"label": "Aux Heat Off Threshold",
+			"description": "Sets the delta from setpoint that stage 3 heating stops. Stage 3 turns off at setpoint + Delta Stage 3.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "17",
+			"label": "Cooling Stage 1 On Threshold",
+			"description": "Sets the delta from setpoint that stage 1 cooling starts.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 7,
+			"defaultValue": 1
+		},
+		{
+			"#": "18",
+			"label": "Cooling Stage 1 Off Threshold",
+			"description": "Sets the delta from setpoint that stage 1 cooling stops. Stage 1 turns off at setpoint - Delta Stage 1.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 6,
+			"defaultValue": 0
+		},
+		{
+			"#": "19",
+			"label": "Cooling Stage 2 On Threshold",
+			"description": "Sets the delta from setpoint that stage 2 cooling starts.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 2,
+			"maxValue": 8,
+			"defaultValue": 2
+		},
+		{
+			"#": "20",
+			"label": "Cooling Stage 1 Off Threshold",
+			"description": "Sets the delta from setpoint that stage 2 cooling stops. Stage 2 turns off at setpoint - Delta Stage 2.",
+			"unit": "°F",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 0
+		},
+		{
+			"#": "21",
+			"label": "Mechanical Status",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 256,
+			"unsigned": true,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "MECH_H1",
+					"value": 1
+				},
+				{
+					"label": "MECH_H2",
+					"value": 2
+				},
+				{
+					"label": "MECH_H3",
+					"value": 4
+				},
+				{
+					"label": "MECH_C1",
+					"value": 8
+				},
+				{
+					"label": "MECH_C2",
+					"value": 16
+				},
+				{
+					"label": "PHANTOM_F",
+					"value": 32
+				},
+				{
+					"label": "MECH_F",
+					"value": 64
+				},
+				{
+					"label": "MANUAL_F",
+					"value": 128
+				},
```

Comment:
Looking at those values, those terribly look like they are supposed to be used in combination as [partial parameters](https://zwave-js.github.io/node-zwave-js/#/config-files/partial-parameters). Not sure if it makes sense though, maybe just one of these states can be active at any given time.

Reply:
Hmm... there's nothing in the manual about these parameters other than the above, in a table. Best I can tell, these are states the device can be in and, for the most part, I believe, only one at a time. For example, you can be in Heating Stage 1, or 2 or 3, or Cooling STage 1 or 2, but not any of those at the same time. I'm assuming that "_F" is Fan, since it can be manual. But I don't know what MECH_F and PHANTOM_F mean, but I would think you could have the fan running when either Heating or Cooling is going on.  But I don't think the lot of them fits the partial parameters pattern.

I have a series of questions for my contact at Ecolink who provided this device to me, but would prefer to get these parameters incorporated so I can experiment with them a bit to see how they work. And maybe refine them later with additional knowledge. Right now, I can't change any of them in the CP (although, I perhaps I could do so through zwave.services?).

Reply:
The zwavejs2mqtt control panel has a field where you provide the param number, value size and value to set parameters. Your explanation makes sense though, we can always change this later if we find that its necessary.

Reply:
Interestingly, for this device I DO NOT have such a field. I have no Config block at all! That's why I can't test any of this via zwave and why i just remembered the services in HA. I haven't tried it yet.

![image](https://user-images.githubusercontent.com/4411963/142294072-b19bf43f-1b0a-4433-b99e-0caec207f20a.png)

Reply:
Just tried using the zwave_js.set_config_parameter service in HA Dev Tools and it failed. I tried targeting both the `sensor.node` and the `climate.xxx` entity. ` Failed to call service: unknown error.`

Reply:
> Interestingly, for this device I DO NOT have such a field

@robertsLando what could cause this? Do you display the Configuration CC tab only when there are values?

Reply:
> Do you display the Configuration CC tab only when there are values?

If it's not shown it means there is no value for that CC